src/lang/wasm.ts

```diff
@@ -336,9 +336,22 @@ export class ProgramMemory {
 }
 
 // TODO: In the future, make the parameter be a KclValue.
-export function sketchGroupFromKclValue(obj: any): SketchGroup | null {
+export function sketchGroupFromKclValue(
+  obj: any,
+  varName: string | null
+): SketchGroup | Error {
   if (obj?.value?.type === 'SketchGroup') return obj.value
-  return null
+  if (!varName) {
+    varName = 'a KCL value'
+  }
+  const actualType = obj?.value?.type
```

Comment:
Should it fall back like this?

```suggestion
  const actualType = obj?.value?.type ?? obj?.type
```